test/runtests.jl

```diff
@@ -835,6 +835,68 @@ end
     end
 
     @testset "Examples" begin
+        @testset "Referencing symbols from other packages" begin
+            @testset "Referencing symbols with import" begin
+                Pkg.add("TimeZones")
```

Comment:
Instead of this, put `TimeZones` in `test/REQUIRE`.

But also I think you should use a simpler package for tests instead, in order to minimize build time and chance for failure. I suggest `Compat`.